backend/core/scripts/minimal-listing.json

```diff
@@ -14,18 +14,13 @@
   },
   "units": [
     {
-      "unitType": {
-        "name": "oneBdrm",
-        "numBedrooms": 1
-      },
+      "unitType": "oneBdrm",
       "status": "available"
     }
   ],
   "assets": [],
   "applicationMethods": [],
   "events": [],
   "displayWaitlistSize": false,
-  "jurisdiction": {
-    "id": "651b272c-aeaa-4808-a581-1c761e52cb53"
-  }
+  "jurisdictionName": "Detroit"
```

Comment:
Has this change propagated to our repo yet or is it still upstream only? Or blocked on #658?

Reply:
This only made it to Exygy's repo in https://github.com/bloom-housing/bloom/pull/1877 (submitted yesterday), and it's not yet in our repo (this current PR aims to do that).